packages/integration-sdk-testing/src/jest.ts

```diff
@@ -238,12 +298,38 @@ export function toMatchGraphObjectSchema<T extends Entity>(
     );
   }
 
-  return {
-    message: () => 'Success!',
-    pass: true,
-  };
+  const hasDistinctKeysResult = toHaveDistinctKeys(received);
+
+  if (hasDistinctKeysResult.pass === true) {
+    return {
+      message: () => 'Success!',
+      pass: true,
+    };
+  } else {
+    return hasDistinctKeysResult;
+  }
+}
+
+function toHaveDistinctKeys(received: { _key: string }[]) {
```

Comment:
This is minor, but I think we could improve this function by not returning a the jest matcher result format and instead return only a boolean. That will make the function more pure and the caller can decide what to do with the resulting boolean. Also I think we can improve the TS typings. Something like the following (untested):

```ts
function graphObjectHasDistinctKeys<
  T extends Entity | ExplicitRelationship
>(received: T[]): boolean {
  const _keys = received.map((r) => r._key);
  return Array.isArray(_keys) && new Set(_keys).size === _keys.length;
}
```

Caller:

```ts
if (graphObjectHasDistinctKeys(received)) {
  return {
    message: () => 'Success!',
    pass: true,
  };
} else {
  return {
    message: () =>
      `Object \`_key\` properties array is not unique: [${_keys}]`,
    pass: false,
  };
}
```

Reply:
Good point. I had originally considered exposing the `toHaveDistinctKeys()` matcher from the SDK but now I'm not sure that makes sense.

Reply:
I'll have to rebase this back off of master (it's branched off of `add-dotenv-config-to-jest-config` which it seems like I'll close without merging). I think it's a good suggestion and worth doing while I'm doing the rebase.